client/src/components/Form/Form.js

```diff
@@ -0,0 +1,56 @@
+import styled from 'styled-components/macro'
+import { useState } from 'react'
+
+const Form = ({ onAddQuestion }) => {
+  const [question, setQuestion] = useState('')
+
+  const changeQuestionHandler = event => {
+    setQuestion(event.target.value)
+  }
+
+  const submitForm = () => {
```

Comment:
You might want to add an `event.preventDefault()` here, so the page doesn't reload!